### DISCUSSION TITLE
Resizing a borderless window does not work with Ubuntu 18

### DISCUSSION THREAD

**Discussion by QuentinWidlocher** [tauri-apps/tauri#2708] [Q&A] [answered]
Hi everyone !

I've seen this issue (https://github.com/tauri-apps/tao/issues/104) on resizing a window without decorations on Windows and I'm wondering why I'm not able to reproduce this behavior on with Ubuntu 18.04.6 LTS.

In my `tauri.conf.json` I have this :

```json
"windows": [
    {
        "title": "Title",
        "width": 800,
        "height": 800,
        "resizable": true,
        "decorations": false,
        "transparent": true
    }
],
```

Minimizing, maximizing etc. works, snapping the window to a side of the screen or another works too, it's just that I cannot select the edges of my window to drag and resize it.

Maybe it's an issue on my side ? Is it working for other people on Ubuntu 18 or more ?

**Comment by amrbashir** [comment, marked as answer]
what's you `tauri info`? Also try clicking in the first few pixels of an edge and drag. The cursor won't change due to a bug that has been resolved and will be available in next release. https://github.com/tauri-apps/wry/pull/402/files#diff-042ccc039d941aaf3412052c4c40c09a990fabbec0177e01d5ab491ebb8929a0R5

**Reply by QuentinWidlocher** [reply]
Ok I'm really sorry, it's working...
The lack of cursor + the really tiny area makes it so difficult to resize that I completely missed it. The area seems smaller than the one on decorated windows no ? Or maybe it's just me ?

Anyway, thanks a lot for your response and for your work !

**Reply by amrbashir** [reply]
> Ok I'm really sorry, it's working... 

Don't be, it is our fault.

> The lack of cursor + the really tiny area makes it so difficult to resize that I completely missed it. The area seems smaller than the one on decorated windows no ? Or maybe it's just me ?

I think they are the same, well at least in our code, we use the same value.
